.readthedocs.yaml

```diff
@@ -11,6 +11,8 @@ build:
   os: "ubuntu-22.04"
   tools:
     python: "3.11"
+  commands:
+    - "python -m pip install ."
```

Comment:
Is it possible to check this version with readthedocs.org?

@cmsirbu

Reply:
We should review RTD options. IIRC it not supports pyproject.toml but not sure if it's worth a full install